user/articles/3-layout-and-design/12-widget-reference.markdown

```diff
@@ -3,160 +3,104 @@
 Liferay Commerce ships with a wide range of widgets you'll use to construct your
 site. This section provides a convenient overview in one place.
 
-## Account Management
```

Comment:
If a table is not an option, what do you think about bullet points, either like this or broken out into 3-4 major sections (groupings of widgets)? I think this is an improvement on skim-ability over a header for each widget, given the short length of each widget's description.